java/archive.gradle

```diff
@@ -1,55 +1,51 @@
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 apply plugin: 'signing'
 
-artifacts {
-    archives javadocJar, sourcesJar
-}
-
-signing {
-    sign configurations.archives
-}
-
-uploadArchives {
-    repositories {
-        mavenDeployer {
-            beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-
-            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-                authentication(userName: ossrhUsername, password: ossrhPassword)
-            }
-
-            snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
-                authentication(userName: ossrhUsername, password: ossrhPassword)
-            }
-
-            pom.project {
-                name 'Scalar Admin'
-                packaging 'jar'
-                // optionally artifactId can be defined here 
-                description 'An admin interface and tool for Scalar services.'
-                url 'https://github.com/scalar-labs/scalar-admin'
-
-                scm {
-                    connection 'scm:git:https://github.com/scalar-labs/scalar-admin.git'
-                    developerConnection 'scm:git:https://github.com/scalar-labs/scalar-admin.git'
-                    url 'https://github.com/scalar-labs/scalar-admin'
-                }
-
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            artifactId = 'scalar-admin'
+            artifact jar
+            artifact javadocJar
+            artifact sourcesJar
+            pom {
+                name = 'Scalar Admin'
+                description = 'An admin interface and tool for Scalar services.'
+                url = 'https://github.com/scalar-labs/scalar-admin'
                 licenses {
                     license {
-                        name 'Apache License, Version 2.0'
-                        url 'http://www.apache.org/licenses/LICENSE-2.0'
+                        name = 'Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0'
                     }
                 }
-
                 developers {
                     developer {
-                        id 'hiroyuki'
-                        name 'Hiroyuki Yamada'
-                        email 'hiroyuki.yamada@scalar-labs.com'
+                        id = 'hiroyuki'
```

Comment:
Can you add your name and Toshi's name as well? 😄

Reply:
> Can you add your name and Toshi's name as well?

@feeblefakie 
Understood.
https://github.com/scalar-labs/scalar-admin/pull/14/commits/41e60fbd7bad0f6967d7dd1c6b8f80c43dba6f78

@brfrn169 
Sorry to bother you.
Could you check if this information is correct?

https://github.com/scalar-labs/scalar-admin/pull/14/commits/41e60fbd7bad0f6967d7dd1c6b8f80c43dba6f78

(I copied it from scalardb but just in case)
Thank you :)